src/test/java/br/dev/marcoalmeida/mb/service/MovieServiceTest.java

```diff
@@ -30,37 +30,29 @@ public class MovieServiceTest {
 
     private MockServerClient mockServerClient;
 
+    public void mockingServerClient(String parameterSearch, String parameterMovie, String fileData) {
+    	mockServerClient
+		    .when(request()
+		        .withQueryStringParameter(parameterSearch, parameterMovie)
+		        .withMethod("GET")
+		    ).respond(response()
+		        .withStatusCode(200)
+		        .withContentType(MediaType.APPLICATION_JSON)
+		        .withBody(fileData)
+		    );
+	}
+    
+    
     @Test
     public void WhenKeywordSupplied_MoviesArePopulated() throws IOException {
-        mockServerClient
-                .when(request()
-                    .withQueryStringParameter("s", "Star")
-                    .withMethod("GET")
-                ).respond(response()
-                    .withStatusCode(200)
-                    .withContentType(MediaType.APPLICATION_JSON)
-                    .withBody(Files.readString(Path.of("src/test/resources/omdb/search.json")))
-                );
+    	String fileData = Files.readString(Path.of("src/test/resources/omdb/search.json"));
```

Comment:
tbm não está errado declarar uma variável intermediária, mas

mockServerClient("s", "Star",Files.readString(Path.of("src/test/resources/omdb/tt0076759.json")))

é mais conciso, 

pergunte-se "vc re-utiliza o conteúdo de fileData em algum lugar?"